### PR TITLE
chore(migration-guide): add migration guide for DateTime and RelativeDateTime

### PR DESCRIPTION
**Pull request by thebesson**
# Purpose of PR

Migration guide for two components: DateTime and RelativeDateTime

## PR Checklist

- [ ] I have read the relevant `readme.md` file(s)
- [ ] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [ ] Tests are added/updated/not required
- [ ] Tests are passing
- [ ] Storybook stories are added/updated/not required
- [ ] Usage notes are added/updated/not required
- [ ] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [ ] Doesn't contain any sensitive information

### CODE REVIEW

File: packages/forma-36-website/src/content/migration-v3-to-v4.mdx [hunk 1]
```diff
@@ -34,6 +34,10 @@ To migrate your project easily, we provided you with a codemod for most of the c
 
 ✅ IconButton: Now you can pass any icon as an element to IconButton component. For more information on how to migrate your IconButton components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-iconbutton-components).
 
+✅ DateTime: You only need to update `format` prop value. For more information on how to migrate your DateTime components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-datetime-components)
+
+✅ RelativeDateTime: No changes, just import new component. For more information on how to migrate your RelativeDateTime components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-datetime-components)
```
Comment (gui-santos) [inline packages/forma-36-website/src/content/migration-v3-to-v4.mdx:39]:
```suggestion
✅ RelativeDateTime: No changes, just import the new component. For more information on how to migrate your RelativeDateTime components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-datetime-components)
```

Just a nitpicky thing, I think we need a "the" over there
But I'm not native speaker or good with grammar 👀

**Review by gui-santos — APPROVED** [review]

**Review by Lelith — APPROVED** [review]
Looks good to me!

File: packages/forma-36-website/src/content/migration-v3-to-v4.mdx [hunk 2]
```diff
@@ -34,6 +34,10 @@ To migrate your project easily, we provided you with a codemod for most of the c
 
 ✅ IconButton: Now you can pass any icon as an element to IconButton component. For more information on how to migrate your IconButton components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-iconbutton-components).
 
+✅ DateTime: You only need to update `format` prop value. For more information on how to migrate your DateTime components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-datetime-components)
```
Comment (alstaicu) [inline packages/forma-36-website/src/content/migration-v3-to-v4.mdx:37]:
```suggestion
✅ DateTime: You only need to update the `format` prop value. For more information on how to migrate your DateTime components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-datetime-components)
```

File: MIGRATION.md [hunk 3]
```diff
@@ -454,6 +458,59 @@ const isCollapsed = true;
 />;
 ```
 
+## DateTime
+
+The API of the DateTime component hasn't changed. The only property that has some updates, is `format` property. From `"FULL"`, `"DATE_ONLY"`, `"TIME_ONLY"`, `"WEEKDAY_DATE"` to `"full"`, `"day"`, `"time"`, `"weekday"`.
```
Comment (alstaicu) [inline MIGRATION.md:463]:
```suggestion
The API of the DateTime component hasn't changed. The `format` property is the only one that was updated: from `"FULL"`, `"DATE_ONLY"`, `"TIME_ONLY"`, `"WEEKDAY_DATE"` to `"full"`, `"day"`, `"time"`, `"weekday"`.
```

File: MIGRATION.md [hunk 4]
```diff
@@ -454,6 +458,59 @@ const isCollapsed = true;
...
+
+<DateTime date="2020-08-17T15:45:00" />
+<DateTime date="2020-08-17T15:45:00" format="day" />
+<DateTime date="2020-08-17T15:45:00" format="weekday" />
+<DateTime date="2020-08-17T15:45:00" format="time" />
+```
+
+### How to migrate your DateTime components
+
+Update `format` value and import. It should change form this:
```
Comment (alstaicu) [inline MIGRATION.md:476]:
```suggestion
Update the `format` value and import. It changes from this:
```

File: MIGRATION.md [hunk 5]
```diff
@@ -454,6 +458,59 @@ const isCollapsed = true;
 />;
 ```
 
+## DateTime
+
+The API of the DateTime component hasn't changed. The `format` property is the only one that was updated: from `"FULL"`, `"DATE_ONLY"`, `"TIME_ONLY"`, `"WEEKDAY_DATE"` to `"full"`, `"day"`, `"time"`, `"weekday"`.
```
Comment (alstaicu) [inline MIGRATION.md:463]:
```suggestion
The API of the DateTime component has not changed. The `format` property is the only one that was updated: from `"FULL"`, `"DATE_ONLY"`, `"TIME_ONLY"`, `"WEEKDAY_DATE"` to `"full"`, `"day"`, `"time"`, `"weekday"`.
```

File: MIGRATION.md [hunk 6]
```diff
@@ -454,6 +458,59 @@ const isCollapsed = true;
...
+
+```tsx static=true
+import { DateTime } from '@contentful/f36-components';
+
+<DateTime date="2020-08-17T15:45:00" format="weekday" />;
+```
+
+## RelativeDateTime
+
+The API of the RelativeDateComponent was not changed.
```
Comment (alstaicu) [inline MIGRATION.md:494]:
```suggestion
The API of the RelativeDateComponent has not changed.
```

File: packages/forma-36-website/src/content/migration-v3-to-v4.mdx [hunk 7]
```diff
@@ -34,6 +34,10 @@ To migrate your project easily, we provided you with a codemod for most of the c
 
 ✅ IconButton: Now you can pass any icon as an element to IconButton component. For more information on how to migrate your IconButton components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-iconbutton-components).
 
+✅ DateTime: You only need to update the `format` prop value. For more information on how to migrate your DateTime components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-datetime-components)
+
+✅ RelativeDateTime: No changes, just import the new component. For more information on how to migrate your RelativeDateTime components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-datetime-components)
```
Comment (alstaicu) [inline packages/forma-36-website/src/content/migration-v3-to-v4.mdx:39]:
```suggestion
✅ RelativeDateTime: There are no changes, just import the new component. For more information on how to migrate your RelativeDateTime components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-datetime-components)
```

File: MIGRATION.md [hunk 8]
```diff
@@ -454,6 +458,59 @@ const isCollapsed = true;
...
+
+### How to migrate your RelativeDateTime components
+
+You just need to update import. From this:
```
Comment (alstaicu) [inline MIGRATION.md:498]: For consistency throughout the guide, should we phrase this as follows?

To migrate RelativeDateComponent to v4, you must update the import. From this:

File: MIGRATION.md [hunk 9]
```diff
@@ -454,6 +458,59 @@ const isCollapsed = true;
...
+
+<DateTime date="2020-08-17T15:45:00" />
+<DateTime date="2020-08-17T15:45:00" format="day" />
+<DateTime date="2020-08-17T15:45:00" format="weekday" />
+<DateTime date="2020-08-17T15:45:00" format="time" />
+```
+
+### How to migrate your DateTime components
+
+Update the `format` value and import. It changes from this:
```
Comment (alstaicu) [inline MIGRATION.md:476]: For consistency throughout the documentation should we phrase this as follows?

To migrate your DateTime component to v4, you must update the `format` value and import. It changes from this:

**Pull request merged**